src/app/child-dev-project/children/model/childSchoolRelation.ts

```diff
@@ -40,13 +40,13 @@ export class ChildSchoolRelation extends TimePeriod {
     description: $localize`:Description of the start date of a relation:The date a child joins a school`,
     anonymize: "retain",
   })
-  start: Date;
+  declare start: Date;
```

Comment:
would [override](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-3.html#override-and-the---noimplicitoverride-flag) instead of declare also work here? (That would feel more explicit and intuitive to me)

Reply:
seems like override does not work for this and this has to do with JavaScript behavior assigning values to class properties:
https://stackoverflow.com/a/66555044/1473411